automatic/mono/tools/chocolateyInstall.ps1

```diff
@@ -1 +1 @@
-﻿Install-ChocolateyPackage 'mono' 'exe' '/SILENT' '{{DownloadUrl}}'
+﻿Install-ChocolateyPackage 'mono' 'msi' '/SILENT' '{{DownloadUrl}}'

```

Comment:
Something tells me this is not going to be `/SILENT` anymore for the arguments since that doesn't work with MSIs.